packages/frontend/src/assets/claim-reward.tsx

```diff
@@ -3,7 +3,7 @@ export function ClaimReward(props: any) {
         <svg
             width="24"
             height="24"
-            viewBox="0 0 24 24"
+            viewBox="0 0 20 20"
```

Comment:
Weird, why did we change this? The viewbox should usually be the same as the width and length to have proper sizing

Reply:
The svg had an empty space that caused it to be misaligned, but at this point we need a new icon.

Reply:
In those cases, instead of doing this, it would be better to use something such as Inkscape and do this the proper way, updating the SVG directly

Reply:
Fixed in bc6d2c0f3e4de1fa660b7ee4992eb05378fb50a8.